source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -347,7 +347,9 @@ export abstract class CardElement {
 
         Utils.setProperty(result, "type", this.getJsonTypeName());
         Utils.setProperty(result, "id", this.id);
-        Utils.setEnumProperty(Enums.HorizontalAlignment, result, "horizontalAlignment", this.horizontalAlignment);
+        if (this.horizontalAlignment !== null) {
```

Comment:
I think this should be `if (this.horizontalAlignment)`

Reply:
I thought that too. But "Left" is equivalent to 0, which is falsey.

Reply:
Want me to shift the values and make "Left" be 1? I normally prefer to have an "Undefined"/"Invalid"/"Uninitialized" zero-equivalent on enums.

Reply:
Indeed